test/gov/hzn_voucher.sh

```diff
@@ -0,0 +1,161 @@
+#!/bin/bash
+
+PREFIX="hzn voucher test:"
+
+echo ""
+echo -e "${PREFIX} Importing and inspecting vouchers with hzn command."
+
+# preparing the voucher file
+cat <<'EOF' > /tmp/sdo_voucher.json
+{"sz":1,"oh":{"pv":113,"pe":1,"r":[1,[4,{"dn":"bruce-dev1.fyre.ibm.com","po":8040,"pow":8040,"pr":"http"}]],"g":"Vmkm97cJRpSHmLfbyMyyCw==","d":"SDO Java Device","pk":[13,1,[91,"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAE1LBVR8bIcRAPcyGHtGBzfk0sJRAQ3pcwBaZo1EDDEHh6NvzmK8BC8DgJfYJj0xL8F3murSIlwK2CHEdOD3TzOA=="]],"hdc":[32,8,"MlpaJw6JrVY+esM8EHfgAKX9c1T8NWZktGb1tuW4CpY="]},"hmac":[32,108,"R72z5AGp30nSCMJN3K9cup2dJ9Uvx6rvIZ/Ydc+h5ts="],"dc":[1,2,[[315,"MIIBNzCB3aADAgECAgYBcYvR8p8wCgYIKoZIzj0EAwIwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoMGEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0yMDA0MTgwNTQ1MjRaFw0yMDA1MTgwNTQ1MjRaMAAwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAAR1o6FQfSCGPjJa6UnnnZwrk2wurYX/7pqyrzjqomddjHkIZIB4hKfKP4DNOBu7Y7OPKiFXTsldlz/0xhEvGf6zMAoGCCqGSM49BAMCA0kAMEYCIQD/MYOBodEL5F4zNewh5vdAEWvVlItLSKLqf35rSSoR/gIhAONzv20dG6to5knATHSaCGMIf08btwbk6w9r0RXs1tYP"],[483,"MIIB3zCCAYWgAwIBAgIUGe+4vTFELPtaQ9zTT2E7tgpb+J4wCgYIKoZIzj0EAwIwRTELMAkGA1UEBhMCQVUxEzARBgNVBAgMClNvbWUtU3RhdGUxITAfBgNVBAoMGEludGVybmV0IFdpZGdpdHMgUHR5IEx0ZDAeFw0xOTEyMjAxOTU2MDNaFw0yMDEyMTkxOTU2MDNaMEUxCzAJBgNVBAYTAkFVMRMwEQYDVQQIDApTb21lLVN0YXRlMSEwHwYDVQQKDBhJbnRlcm5ldCBXaWRnaXRzIFB0eSBMdGQwWTATBgcqhkjOPQIBBggqhkjOPQMBBwNCAATUsFVHxshxEA9zIYe0YHN+TSwlEBDelzAFpmjUQMMQeHo2/OYrwELwOAl9gmPTEvwXea6tIiXArYIcR04PdPM4o1MwUTAdBgNVHQ4EFgQUJcffU8efi7VUyZ3NHZ7MXAIkbYYwHwYDVR0jBBgwFoAUJcffU8efi7VUyZ3NHZ7MXAIkbYYwDwYDVR0TAQH/BAUwAwEB/zAKBggqhkjOPQQDAgNIADBFAiEAjfCAWcXdHtLp837WftNVZhSvb1mrNTMNLGcopa+4zt0CIHdusGmDHTPcIk/lq1m91nKE4XCGUI7wD4V3zk5Kpb0r"]]],"en":[{"bo":{"hp":[32,8,"gzGJRwRWVT0IiK3xGaTo68LwK3juTdeEzzS+Tfsf7dg="],"hc":[32,8,"jwqcp/laj60ScXyQ843Czz2ilbYfLw3gfnoRFBWHNn8="],"pk":[13,1,[91,"MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEWVUE2G0GLy8scmAOyQyhcBiF/fSUd3i/Og7XDShiJb2IsbCZSRqt1ek15IbeCI5z7BHea2GZGgaK63cyD15gNA=="]]},"pk":[0,0,[0]],"sg":[71,"MEUCICKiSEc05cHU3nQcr+lhN+gpCWUkjJlk/iexds/1TpqpAiEAnD9WaJo8P1WhSBZHvzFiyk0A0DneW6NQqm+bLhnjezQ="]}]}
```

Comment:
If the IBM specific hostname is not critical to the test, can you change it to something generic?

Reply:
fixed